lib/active_merchant/billing/credit_card.rb

```diff
@@ -133,6 +133,16 @@ def brand=(value)
         @brand = (value.respond_to?(:downcase) ? value.downcase : value)
       end
 
+      # Sets if the credit card was build with GooglePay.
+      attr_accessor :google_pan_only
+
+      # Returns if this credit card was build with GooglePay.
+      #
+      # Returns [Boolean]
```

Comment:
I'm not sure if this is used in a technical way, but best to keep the pattern:
```suggestion
      # @return [Boolean]
```